custom_components/tahoma/cover_devices/tahoma_cover.py

```diff
@@ -103,20 +102,20 @@ async def async_set_cover_tilt_position(self, **kwargs):
     def is_closed(self):
         """Return if the cover is closed."""
 
-        if self.current_cover_position is not None:
-            return self.current_cover_position == 0
-
         state = self.executor.select_state(
             CORE_OPEN_CLOSED_STATE,
             CORE_SLATS_OPEN_CLOSED_STATE,
             CORE_OPEN_CLOSED_PARTIAL_STATE,
             CORE_OPEN_CLOSED_PEDESTRIAN_STATE,
             CORE_OPEN_CLOSED_UNKNOWN_STATE,
-            MYFOX_SHUTTER_STATUS_STATE,
```

Comment:
Why did you remove this one? If I recall correctly, we use this for the Camera Shutter.